warehouse/models/mart/gtfs_quality/fct_daily_gtfs_dataset_guideline_checks.md

```diff
@@ -0,0 +1,32 @@
+{% docs fct_daily_gtfs_dataset_guideline_checks %}
+
+Each row represents a date/guideline check/gtfs dataset combination, with pass/fail
+information indicating whether that feed complied with that check on that date.
+Only contains checks that are performed at the gtfs_dataset
```

Comment:
Can we clarify documentation for which are schedule-only vs. both? Technically that is kind of mixing grains here but I think it's probably ok

Reply:
Clarified a bit!